### ISSUE TITLE
Field can have an empty value

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#13244]
If you try to create a field with the name : '重要度' it will insert a field into the db that has an empty field_variable (0 length string).  We need to prevent this in the api

```
# select field_name,velocity_var_name from field where structure_inode ='7002bbc6-2182-4f94-8e89-e237849f2c5a';
             field_name             |        velocity_var_name
------------------------------------+---------------------------------
 重要度                             |
```

**Comment by john-thomas-dotcms** [comment]
So more generally, how should the UI handle it when a velocity variable name (probably of a minimum length, like 3 or 5 characters) can't be created from the field name?

Just preventing it would mean that users can't create field names with all non-English characters (which might not be a good thing).  So would it make sense that for any field name which results in a velocity var name less than X characters, pre- or post-fix the field type (and maybe a unique number) to the variable name?

**Comment by wezell** [comment]
fixed with this commit 12cd4ab99aef285365f52a7a92f5abb5e22af345
